project_event/views/project_project_view.xml

```diff
@@ -60,6 +60,8 @@
                         </group>
                         <group>
                             <field name="partner_id"/>
+                            <field name="client_type"/>
+                            <field name="sector_id"/>
```

Comment:
@jananjoy the sector field should be visible only if the client_type is not empty

Reply:
@rimbendhaou Done in both project_Project and Project_Task too